Apps/Admin/Server/Controllers/BroadcastController.cs

```diff
@@ -94,5 +93,22 @@ public async Task<RequestResult<Broadcast>> UpdateBroadcast(Broadcast broadcast)
         {
             return await this.broadcastService.UpdateBroadcastAsync(broadcast).ConfigureAwait(true);
         }
+
+        /// <summary>
+        /// Deletes the Broadcast that is passed to the controller.
+        /// </summary>
+        /// <param name="broadcast">The broadcast object to delete.</param>
+        /// <returns>The broadcast object delete wrapped in a Request result.</returns>
+        /// <response code="200">Returns the communication json of the deleted object wrapped in a request result.</response>
```

Comment:
Recommend updating these 2 lines in the documentation to more closely match the other methods in the controller; the documentation in CommunicationController is a little confusing.